jwst/jump/jump_step.py

```diff
@@ -15,35 +15,46 @@ class JumpStep(Step):
     """
 
     spec = """
+        [Baseline Jump Detection Parameters]
```

Comment:
Unfortunately I've discovered that this kind of entry causes the text within the brackets to be a new hierarchy within the param names themselves, i.e. the following params would become "BaselineJumpDetectionParameters.rejection_threshold", and so on. Which is not a good thing. So we'll need to remove these and just live with the long, single list of param names. At least they're now grouped in the docs, which helps.

Reply:
@mwregan2 These [] entries still need to be removed. They don't work as we had hoped.